src/main/java/cloud/fogbow/ras/core/models/orders/Order.java

```diff
@@ -170,6 +183,47 @@ public void setRequirements(Map<String, String> requirements) {
         this.requirements = requirements;
     }
 
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public String getUserId() {
+        return this.userId;
+    }
+
+    public void setSerializedSystemUser(String serializedSystemUser) {
+        this.serializedSystemUser = serializedSystemUser;
+    }
+
+    public String getSerializedSystemUser() {
+        return this.serializedSystemUser;
+    }
+
+    public void setProviderId(String providerId) {
+        this.providerId = providerId;
+    }
+
+    public String getProviderId() {
+        return this.providerId;
+    }
+
+    public void serializeSystemUser() {
+        SerializableSystemUser serializableSystemUser = new SerializableSystemUser(this.getSystemUser());
+        this.setSerializedSystemUser(GsonHolder.getInstance().toJson(serializableSystemUser));
+        this.setUserId(this.getSystemUser().getId());
+        this.setProviderId(this.getSystemUser().getIdentityProviderId());
+    }
+
+    @PostLoad
+    private void deserializeSystemUser() {
+        try {
+            SerializableSystemUser serializableSystemUser = GsonHolder.getInstance().fromJson(this.getSerializedSystemUser(), SerializableSystemUser.class);
+            this.setSystemUser(serializableSystemUser.getSystemUser());
+        } catch(ClassNotFoundException exception) {
+            exception.printStackTrace();
```

Comment:
Standard output might be lost if the RAS process runs as a Daemon. It's better to have a Logger instance here and call Logger.error(). Take a look at ApplicationFacade to see how to do this.